packages/client/rubygem-foreman_scap_client/rubygem-foreman_scap_client.spec

```diff
@@ -3,32 +3,49 @@
 
 %define rubyabi 1.8
```

Comment:
Is this still used?

Reply:
on el6, yes, but we don't build for that, so can be cleaned up I guess

Reply:
We still have other el6 macros, so we either clean them all up or leave them all in.